packages/plugin/altair/src/types.ts

```diff
@@ -2,11 +2,21 @@ import type { PickRequired } from 'graphql-ez';
 import type { RenderOptions } from 'altair-static-slim';
 import type { IncomingMessage, ServerResponse } from 'http';
 
-export interface AltairOptions extends RenderOptions {
+export interface AltairOptions extends Omit<RenderOptions, 'baseURL' | 'endpointURL'> {
   /**
    * @default "/altair"
    */
   path?: string;
+
+  /**
+   * URL to be used as a base for relative URLs
+   */
+  base?: string;
```

Comment:
I feel like this specific change is not needed at all, baseURL is altair specific and doesn't make sense to remove the "URL" just for no real reason

Reply:
I was unsure about both of them, but opted for the unifying approach as both are part of the user-facing interface. I would say it's a matter of taste if one opts for the consistency with graphql-ez or with altair. You only want  to have`base`, but not `endpoint`, removed, right?

Reply:
baseURL doesn't ever need to be manually specified, just specifying "path" is enough, baseURL doesn't need to be highlighted

Reply:
yeah, only base, aligning endpoint is valuable